gnovm/stdlibs/errors/errors_test.gno

```diff
@@ -51,3 +51,60 @@ func ExampleNew_errorf() {
 	}
 	// Output: user "bimmler" (id 17) not found
 }
+
+type customError struct {
+	msg string
+}
+
+func (e *customError) Error() string {
+	return e.msg
+}
+
+func (e *customError) Is(target error) bool {
+	t, ok := target.(*customError)
+	if !ok {
+		return false
+	}
+	return e.msg == t.msg
+}
+
+func TestIs(t *testing.T) {
```

Comment:
can we have same set of tests as [`errors/wrap.go:L16-69`]?(https://cs.opensource.google/go/go/+/refs/tags/go1.22.2:src/errors/wrap_test.go;l=16-69)